src/support.js

```diff
@@ -85,6 +85,33 @@ function loadDynamicLibrary(lib) {
 }
 
 #if WASM
+var asm2wasmImports = { // special asm2wasm imports
```

Comment:
i think this needs to be higher up - it's inside an ifdef for relocatable, but we need this for all wasm code (and causes the CI failures)